README.md

```diff
@@ -97,6 +97,8 @@ You will need the following components to get started with Skaffold:
     cd examples/getting-started
     ```
 
+1. In the skaffold.yaml file, update the `build.artifacts.imageName` stanza to the registry where your image will be pushed (`<registry/image:tag>`). For example `gcr.io/<your-project-ID>/your-image`, where `gcr.io/<your-project-ID>` is your GCP container registry location. As mentioned earlier, your docker client should be configured to push to this location.
```

Comment:
These instructions are actually for "local clusters" (i.e. minikube or docker for desktop) where you don't actually have to do this step. Maybe we should make the heading or tutorial clearer, since a few users have run into this while trying it out on GKE.

Reply:
Given that clarification I'd close this issue. It's more of a UX thing, it's not immediately intuitive to look in skaffold.yaml for the resultant image location/name.